src/charm.py

```diff
@@ -88,8 +110,8 @@ def _install_cli_resources(self):
             self.unit.status = BlockedStatus("CLI resources missing.")
             log.error(e)
             return
-        except PermissionError as e:
-            log.error(f"Cannot copy CLI binaries {e}")
+        except Exception as e:
+            log.error(f"CLI binaries could not be installed: {e}")
```

Comment:
whew this is broad.  Perhaps the `try...except`  in `_install_cli_resources` should be broken up a bit